db/seeds.rb

```diff
@@ -9,25 +9,49 @@
 # end
 
 STAR_COLUMNS = ["pl_hostname","ra","dec","st_dist","st_teff","st_mass","st_rad","st_spstr","st_logg","st_lum","st_dens","st_age"]
+PLANET_COLUMNS = ["pl_name","pl_tranflag","pl_eqt","pl_discmethod","pl_orbper","pl_orbeccen","pl_massj","pl_radj","pl_dens"]
```

Comment:
Line is too long. [124/80]<br>Freeze mutable objects assigned to constants.<br>Space missing after comma.<br>Use `%w` or `%W` for an array of words.